CRM/Extendedreport/Form/Report/ExtendedReport.php

```diff
@@ -2330,7 +2330,7 @@ function extractFieldsAndFilters($field, $fieldName, &$filter) {
             'Multi-Select',
```

Comment:
@jmcclelland these lines can go then? (given any release I cut will be for civicrm 5.28 or greater)

Reply:
Which lines? Sorry, I don't follow. Is 'Multi-Select' deprecated?

Reply:
Oh, I think you are referring to adv-multi-select, right?